vendor.conf

```diff
@@ -48,4 +49,3 @@ github.com/opencontainers/selinux master
 golang.org/x/sys master
 github.com/tchap/go-patricia v2.2.6
 github.com/BurntSushi/toml master
-github.com/pquerna/ffjson d49c2bc1aa135aad0c6f4fc2056623ec78f5d5ac
```

Comment:
And this removed item also does not correspond to removing any files from `vendor`?!